$OpenBSD: patch-src_common_h,v 1.9 2013/05/24 08:31:37 dcoppa Exp $

Make --volume work with and without softvol enabled
(upstream svn revision r2426)

--- src/common.h.orig	Mon Feb 18 15:58:47 2013
+++ src/common.h	Thu May 23 15:20:45 2013
@@ -310,6 +310,7 @@ gboolean softvol;
 gboolean remember_softvol;
 gdouble volume_softvol;
 gint volume_gain;
+gint pref_volume;
 gint verbose;
 gint autostart;
 //gint actual_x, actual_y;
